dingo/gw/noise/synthetic/generate_dataset.py

```diff
@@ -113,11 +113,11 @@ def generate_dataset(real_dataset, settings: Dict, num_samples, num_processes: i
             psds[:, real_dataset.domain.min_idx : real_dataset.domain.max_idx + 1]
         )
 
-    synthetic_dataset_dict["parameters"] = parameters_dict
+    synthetic_dataset_dict["asd_parameterizations"] = parameters_dict
     synthetic_dataset_dict["asds"] = asds_dict
 
```

Comment:
Does this dict contain the GPS times here? If so, are they not meaningless at this point? Should they be del'd? (I don't know if they would cause problems with the ASDDataset class, so it's up to you.

```suggestion
        del synthetic_dataset_dict["gps_times"]

```

Reply:
Yeah, it's copied from the initial dataset. I don't think they are generally meaningless, since 
a) we might only parameterise/smoothen the PSDs, in which case the gps_times still correspond to the psds and b) it may still be good to know what data was used for generating a synthetic dataset. But if you insist, I can delete it